.github/workflows/ci.yml

```diff
@@ -234,7 +234,7 @@ jobs:
         # To avoid problems with non-relocatable packages, we unzip the archive exactly in the same directory
         # that has been used to create the pre-compiled archive
         cd C:/
-        wget https://github.com/robotology/robotology-superbuild-dependencies-vcpkg/releases/download/v0.3.0/vcpkg-robotology-with-gazebo.zip
+        wget https://github.com/robotology/robotology-superbuild-dependencies-vcpkg/releases/download/v0.5.0/vcpkg-robotology-with-gazebo.zip
```

Comment:
Do you prefer to control the version, or we could always download the latest?

Reply:
I think it is important to control the version. I prefer to open a PR for any new release of `robotology-superbuild-dependencies-vcpkg` to explicitly check that the new release works fine, rather then do a release of  `robotology-superbuild-dependencies-vcpkg` that may break the robotology-superbuild CI, and then be force to a rollback PR to robotology-superbuild to fix the problem. As we use the latest archive in the docs, if the latest does not work correctly it is still a problem that needs to address fast, but I prefer not to break CI that would basically block any other PR from being merged without forcing.

Reply:
Makes sense!